src/app/nft/nft.service.ts

```diff
@@ -0,0 +1,23 @@
+import {Injectable, Injector } from '@angular/core';
+import { RestService } from '@lagoshny/ngx-hal-client';
+import { Observable } from 'rxjs/internal/Observable';
+
+import { NFT } from '../login-basic/nft';
+
+
+@Injectable()
+export class NftService extends RestService<NFT>{
+
+  constructor(injector: Injector) {
+    super(NFT, 'nfts', injector );
```

Comment:
The second parameter in this call must be a string matching the URI name in the API: 'nFTs'.